packages/@aws-cdk/cx-api/lib/cloud-assembly.ts

```diff
@@ -369,6 +369,8 @@ export interface AssemblyBuildOptions {
   /**
    * Include the specified runtime information (module versions) in manifest.
    * @default - if this option is not specified, runtime info will not be included
+   * @deprecated All template modifications that should result from this should
+   * have already been inserted into the template.
    */
   readonly runtimeInfo?: RuntimeInfo;
```

Comment:
Do we really want to continue to emit this info into the assembly? I think this should now be always `undefined`

Reply:
Yes. But I don't think we can remove it from this API due to backwards compatibility guarantees.